src/assets/translations/en.js

```diff
@@ -64,4 +63,4 @@ export default {
     "videoInfo": "About This Video",
     "volume": "Volume",
     "volumeSlider": "Volume"
-};
+};
```

Comment:
?

Reply:
The changes were made by running the new CSV file through the translationParser script. It passed the karma tests/linter so I didn't look at the code. fixing now.

Reply:
A line was removed, so it's detecting that the last line is now on 66 instead of 67. I don't know why it's marked as a change.

Reply:
This is because there used to be a white space (empty line) at the end of the file. Our linter looks for an empty space at the end of each file

Reply:
Thanks for the heads up! I updated it.